jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/SlideGroupAssert.java

```diff
@@ -34,4 +37,17 @@ public SlideGroupAssert visibleSlidesPosition(int position) {
         jdiAssert(element().slidesPosition(), Matchers.is(String.format("transform: translateX(%spx);", position)));
         return this;
     }
+
+    @JDIAction("Assert that '{name}' has center active slide")
+    public SlideGroupAssert centerActiveSlide() {
+        int lstSliderSize = element().getAllSlide().size();
+        if (element().previousButtonIsDisabled()) {
+            jdiAssert(element().position() <= Math.round((double) lstSliderSize / 2), Matchers.is(true));
+        } else if (element().nextButtonIsDisabled()) {
+            jdiAssert(element().position() > lstSliderSize / 2, Matchers.is(true));
```

Comment:
this logic is not checked in your test. Need to improve test steps. 
think to use Math.round((double) lstSliderSize / 2 instead of lstSliderSize / 2